plugins/net_plugin/include/eosio/net_plugin/auto_bp_peering.hpp

```diff
@@ -205,12 +205,12 @@ class bp_connection_manager {
       return conn->bp_connection == Connection::bp_connection_type::non_bp && conn->socket_is_open() && conn->incoming_and_handshake_received();
```

Comment:
Need to revert bn256 submodule change.

Reply:
Yes, I checked this in by mistake.
But actually, could you catchup with main in your branch? That would also resolve it.

Reply:
Updated my branch